app/src/main/java/org/hackillinois/android/view/schedule/EventsAdapter.kt

```diff
@@ -7,7 +7,11 @@ import androidx.recyclerview.widget.RecyclerView
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import kotlinx.android.synthetic.main.schedule_event_list_item.view.*
+import kotlinx.android.synthetic.main.activity_event_info.view.*
+import kotlinx.android.synthetic.main.activity_event_info.view.eventLocation
+import kotlinx.android.synthetic.main.activity_event_info.view.eventTitle
+import kotlinx.android.synthetic.main.event_tile.view.*
+//import kotlinx.android.synthetic.main.schedule_event_list_item.view.*
```

Comment:
Can you delete this line?

Reply:
Oh sorry I forgot we shouldn't commit commented code. Thanks.